### PR TITLE
Update README.md to highlight supported distros and languages

### PR DESCRIPTION
**Pull request by zhill**
Same content, just bullet points instead of a single item for each type. Just visually easier to identify if you're looking for this info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
 
 **Features**
 - Scan the contents of a container image or filesystem to find known vulnerabilities.
-- Find vulnerabilities for major operating system packages across Alpine, BusyBox, CentOS / Red Hat, and Debian / Ubuntu flavored distributions
-- Find vulnerabilities for Ruby (Bundler), Java, JavaScript (npm/yarn), Python (Egg/Wheel) packages, and Python Pip/requirements.txt listings
+- Find vulnerabilities for major operating system packages
+  - Alpine,
```
Comment (alfredodeza) [inline README.md:15]:
```suggestion
  - Alpine
```

File: README.md [hunk 2]
```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
 
 **Features**
 - Scan the contents of a container image or filesystem to find known vulnerabilities.
-- Find vulnerabilities for major operating system packages across Alpine, BusyBox, CentOS / Red Hat, and Debian / Ubuntu flavored distributions
-- Find vulnerabilities for Ruby (Bundler), Java, JavaScript (npm/yarn), Python (Egg/Wheel) packages, and Python Pip/requirements.txt listings
+- Find vulnerabilities for major operating system packages
+  - Alpine, 
+  - BusyBox
+  - CentOS / Red Hat
+  - Debian / Ubuntu flavored distributions
```
Comment (alfredodeza) [inline README.md:18]: Are there Ubuntu flavored distros? I think there are a couple but usually people refer to Debian-based distros? (this is fine to ignore)

```suggestion
  - Debian, and Debian flavored distributions
```

File: README.md [hunk 3]
```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
...
+- Find vulnerabilities for language-specific packages
+  - Ruby (Bundler), 
+  - Java (jars, etc), 
+  - JavaScript (npm/yarn), 
+  - Python (Egg/Wheel) packages
```
Comment (alfredodeza) [inline README.md:23]: These are all packages

```suggestion
  - Python (Egg/Wheel)
```

File: README.md [hunk 4]
```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
...
 - Scan the contents of a container image or filesystem to find known vulnerabilities.
-- Find vulnerabilities for major operating system packages across Alpine, BusyBox, CentOS / Red Hat, and Debian / Ubuntu flavored distributions
-- Find vulnerabilities for Ruby (Bundler), Java, JavaScript (npm/yarn), Python (Egg/Wheel) packages, and Python Pip/requirements.txt listings
+- Find vulnerabilities for major operating system packages
+  - Alpine, 
+  - BusyBox
+  - CentOS / Red Hat
+  - Debian / Ubuntu flavored distributions
+- Find vulnerabilities for language-specific packages
+  - Ruby (Bundler),
```
Comment (alfredodeza) [inline README.md:20]:
```suggestion
  - Ruby (Bundler)
```

File: README.md [hunk 5]
```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
...
-- Find vulnerabilities for major operating system packages across Alpine, BusyBox, CentOS / Red Hat, and Debian / Ubuntu flavored distributions
-- Find vulnerabilities for Ruby (Bundler), Java, JavaScript (npm/yarn), Python (Egg/Wheel) packages, and Python Pip/requirements.txt listings
+- Find vulnerabilities for major operating system packages
+  - Alpine, 
+  - BusyBox
+  - CentOS / Red Hat
+  - Debian / Ubuntu flavored distributions
+- Find vulnerabilities for language-specific packages
+  - Ruby (Bundler), 
+  - Java (jars, etc),
```
Comment (alfredodeza) [inline README.md:21]:
```suggestion
  - Java (jars, etc)
```

File: README.md [hunk 6]
```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
...
-- Find vulnerabilities for Ruby (Bundler), Java, JavaScript (npm/yarn), Python (Egg/Wheel) packages, and Python Pip/requirements.txt listings
+- Find vulnerabilities for major operating system packages
+  - Alpine, 
+  - BusyBox
+  - CentOS / Red Hat
+  - Debian / Ubuntu flavored distributions
+- Find vulnerabilities for language-specific packages
+  - Ruby (Bundler), 
+  - Java (jars, etc), 
+  - JavaScript (npm/yarn),
```
Comment (alfredodeza) [inline README.md:22]:
```suggestion
  - JavaScript (npm/yarn)
```

**Comment by wagoodman** [comment]
@alfredodeza I incorporated your comments + a few MD linting errors

**Pull request merged**